marlon/baseline_models/env_wrappers/attack_wrapper.py

```diff
@@ -196,21 +198,29 @@ def step(self, action: Action) -> Tuple[Observation, float, bool, Dict[str, Any]
 
         observation, reward, done, info = self.cyber_env.step(translated_action)
         transformed_observation = self.transform_observation(observation)
+        self.cyber_rewards.append(reward)
+
         if done:
             logging.warning("Attacker Won")
+
         self.timesteps += 1
-        if self.reset_request or self.timesteps > self.max_timesteps:
+        if self.reset_request:
+            done = True
+
+        if self.timesteps > self.max_timesteps:
+            reward_modifier += LOSS_PENALTY
```

Comment:
There are preset penalties in cyber_env. They are defaulted to 5000 and 0 respectively. I think we should use these instead of hard coding numbers into the wrappers. Maybe it can be a parameter for the universe build method?

Reply:
Did that and took the opportunity to add the agent loading, so we can now evaluate saved agents.